docs/sphinx/user/inputs_Momentum_Sources.rst

```diff
@@ -160,3 +160,87 @@ Section: Momentum Sources
    the uniform_timetable body force type and is only active for the uniform_timetable type.  Note that the code 
    expects there to be a single-line header at the beginning of the uniform timetable file; if no header exists, 
    the first line of data will be ignored.
+
+.. input_param:: DragForcing.drag_coefficient
+
+   **type:** Real, optional
+
+   This value specifies the coefficient for the forcing term in the immersed boundary forcing method. It is currently
+   recommended to use the default value to avoid initial numerical stability. 
+
+.. input_param:: DragForcing.sponge_strength
+
+   **type:** Real, optional
+
+   The value of the sponge layer coefficient. It is recommended to use the default value of 1.0.  
+
+.. input_param:: DragForcing.sponge_density
+
+   **type:** Real, optional
+
+   The value of the sponge layer density. It is recommended to use the default value of 1.0.  
+
+.. input_param:: DragForcing.sponge_distance_west
+
+   **type:** Real, optional
+
+   This value is specified as a negative value when the inflow x-velocity is <=0. The default value is -1000 m and can be changed if strong 
+   reflections are observed. 
+
+.. input_param:: DragForcing.sponge_distance_east
+
+   **type:** Real, optional
+
+   This value is specified as a positive value when the inflow x-velocity is >=0. The default value is 1000 m and can be changed if strong 
+   reflections are observed. 
+
+.. input_param:: DragForcing.sponge_distance_south
+
+   **type:** Real, optional
+
+   This value is specified as a negative value when the inflow y-velocity is <=0. The default value is -1000 m and can be changed if strong 
+   reflections are observed. 
+
+.. input_param:: DragForcing.sponge_distance_north
+
+   **type:** Real, optional
+
+   This value is specified as a positive value when the inflow y-velocity is >=0. The default value is 1000 m and can be changed if strong 
+   reflections are observed. 
+
+.. input_param:: DragForcing.sponge_west
+
+   **type:** int, optional
+
+   This term turns on the sponge layer in the east (-x) boundary. The default value is 0. 
```

Comment:
These all say "east". Presumably that's not right ;)

Reply:
I think have fixed most.